src/esp_loader.ts

```diff
@@ -917,12 +918,16 @@ export class ESPLoader extends EventTarget {
       let words = unpack("I".repeat(Math.floor(data.length / 4)), data);
       let nextReg = SPI_W0_REG;
 
-      this.logger.debug(`Words Length: ${words.length}`);
+      if (this.debug) {
+        this.logger.debug(`Words Length: ${words.length}`);
```

Comment:
It's interesting we have both `this.debug` and I bet that `this.logger.debug` has it's own filtering … 🤷